source/sass/blocks/header.scss

```
@@ -1,13 +1,20 @@
 .header {
-  background-color: $color-primary;
+  background-color: rgba(0, 0, 0, 0.3);
```

Даже в таком случае можно использовать переменную. В этом как раз преимущество препроцессора, он ее "развернет" в нужный формат:
```suggestion
  background-color: rgba($color-black, 0.3);
```